mtp_cashbook/templates/cashbook/transaction_batch_list.html

```diff
@@ -1,6 +1,8 @@
 {% extends "base.html" %}
 {% load currency core i18n %}
 
+{% block template_type %}v-batch{% endblock %}
```

Comment:
Is this shorthand used much elsewhere? Is there a reason we can't use `view-batch` (or even `batch-view`)?